build.assets/Makefile

```diff
@@ -577,3 +577,16 @@ build-multiarch:
 		--build-arg GID=$(shell id -g) \
 		--tag $(BUILDBOX) \
 		--file Dockerfile-multiarch .
+
+.PHONY:generate-eventschema
```

Comment:
I initially made it run in grpcbox but @codingllama  identified this would make grpc building too slow. I'll create a toolingbox with all tools from build.assets/tooling. However this PR is already XXL, if that's OK with you I'll address this point in a subsequent PR.